src/__tests__/testUtils.ts

```diff
@@ -41,3 +41,31 @@ test('createErrorResponse', () => {
     },
   });
 });
+
+export function waitSeconds(seconds: number) {
```

Comment:
Can be removed - we already use `sleep-promise`: https://github.com/cognitedata/cognitesdk-js/blob/f1/0.6/src/__tests__/autoPagination.spec.ts#L3